Core/OfficeDevPnP.Core/Framework/Provisioning/ObjectHandlers/ObjectTeams.cs

```diff
@@ -622,66 +622,68 @@ private static bool SetTeamChannels(PnPMonitoredScope scope, TokenParser parser,
                     if (channelId == null) return false;
 
                     // If there are any Tabs for the current channel
-                    if (channel.Tabs == null || !channel.Tabs.Any()) continue;
-
-                    foreach (var tab in channel.Tabs)
+                    if (channel.Tabs.Count > 0)
```

Comment:
Same here, the foreach over an empty collection will simply skip it. I think we don't need to test Count > 0.